models/User.js

```diff
@@ -24,10 +24,20 @@ User.init(
         len: [3, 30], // Example length validation
       },
     },
-    firstName: {
+    firstname: {
       type: DataTypes.STRING,
       allowNull: false,
     },
+    location: {
+      type: DataTypes.STRING,
+      allowNull: false,
```

Comment:
Do we need to set this to allowNull: false?  I think the user should be able to opt out of sharing their location if they desire.

Reply:
I'm fine with changing that field. I took off the allownull, created another test user and it looks to not affect the signup process now. 

Side note, I also realized I needed to unhide the redirection on user creation, so that was just added on the PR as well.